<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sort</class>
 <widget class="QMainWindow" name="Sort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>579</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sort</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="SortAlgSelect">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bubble sort</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="colCount">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="delay">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCount">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of columns </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDelay">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MixBut">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>190</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Mix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SortBut">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>230</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopBut">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>270</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>⏸</string>
    </property>
   </widget>
   <widget class="QLabel" name="Comparisons">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>400</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Comparisons: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Swaps">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Swaps: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="QuitBut">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
